devito/passes/iet/misc.py

```diff
@@ -57,13 +57,30 @@ def hoist_prodders(iet):
 @iet_pass
 def relax_incr_dimensions(iet, **kwargs):
     """
-    Recast Iterations over IncrDimensions as ElementalFunctions; insert
-    ElementalCalls to iterate over the "main" and "remainder" regions induced
-    by the IncrDimensions.
+    This pass adjusts the bounds of blocked loops in order to
+    include the "remainder regions". The Iterations in the input IET span
```

Comment:
I would drop the whole sentence "The Iterations in the [..] thus missing the remainder part"